blueoil/templates/lmnet/classification.tpl.py

```diff
@@ -78,7 +78,34 @@
 
 NETWORK = EasyDict()
 NETWORK.OPTIMIZER_CLASS = tf.train.MomentumOptimizer
-NETWORK.OPTIMIZER_KWARGS = {"momentum": 0.9, "learning_rate": {{learning_rate}}}
+
+NETWORK.OPTIMIZER_KWARGS = {"momentum": 0.9}
```

Comment:
I agree and modified.

Reply:
@lm-konda 
Thanks,  but sorry, I was misleading 😭 

* When `lr_setting` is `fixed`, the parameter of `NETWORK.OPTIMIZER_KWARGS` is overridden here.
https://github.com/blue-oil/blueoil/blob/b7c93dc0c8a9075ddb03ac04bc12f28eeb94f6c1/blueoil/templates/lmnet/classification.tpl.py#L103-L104
* Only when `lr_setting` is not `fixed`, We use `NETWORK.OPTIMIZER_KWARGS = {"momentum": 0.9}`.

In order to avoid override `NETWORK.OPTIMIZER_KWARGS`,  remove the `NETWORK.OPTIMIZER_KWARGS = {"momentum": 0.9}` line 82 and add like this
```
if '{{learning_rate_setting}}' != 'fixed':
    NETWORK.OPTIMIZER_KWARGS = {"momentum": 0.9}
    NETWORK.LEARNING_RATE_FUNC = tf.train.piecewise_constant
```

It is easy for me to read the source code, How about this?

Reply:
Thank you! I understand and simplify this.